extras/cloudsql/README.md

```diff
@@ -63,8 +63,8 @@ kubectl create secret -n $NAMESPACE generic cloud-sql-admin \
 This command will also deploy two Kubernetes Jobs, to populate the accounts and ledger dbs with Tables and test data.
 
 ```
-kubectl apply -n $NAMESPACE -f ./populate-jobs
-kubectl apply -n $NAMESPACE -f ./kubernetes-manifests
+kubectl apply -n ${NAMESPACE} -f ./populate-jobs
+kubectl apply -n ${NAMESPACE} -k ./kubernetes-manifests
```

Comment:
Note that the manifests in `./populate-jobs` will not be touched and will not use Kustomize.

Reply:
Reason: It seems like the original author of this tutorial wanted to keep the "./populate-jobs" separate — so it's clearer (explicit) to users what's happening.